Dockerfile

```diff
@@ -54,7 +54,7 @@ RUN mkdir -p /home/app/webapp/log && \
 USER app
 
 # precompile the Rails assets
-RUN RAILS_ENV=production bundle exec rails assets:precompile
+RUN RAILS_ENV=development bundle exec rails assets:precompile
```

Comment:
Could I check why this change is necessary? It will impact the docker-compose installation

Reply:
Sorry about that! I was testing my local docker compose! I must've accidentally added this commit. Will remove right away.